sw/host/opentitanlib/src/ownership/isfb.rs

```diff
@@ -0,0 +1,155 @@
+// Copyright lowRISC contributors (OpenTitan project).
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::Result;
+use byteorder::{LittleEndian, ReadBytesExt, WriteBytesExt};
+use serde::{Deserialize, Serialize};
+use serde_annotate::Annotate;
+use std::io::{Read, Write};
+
+use super::misc::{TlvHeader, TlvTag};
+use super::GlobalFlags;
+
+/// The owner Integration Specific Firmware Binding (ISFB) configuration
+/// describes the configuration parameters for the ISFB region.
+#[derive(Debug, Serialize, Deserialize, Annotate)]
+pub struct OwnerIsfbConfig {
+    /// Header identifying this struct.
+    #[serde(
+        skip_serializing_if = "GlobalFlags::not_debug",
+        default = "OwnerIsfbConfig::default_header"
+    )]
+    pub header: TlvHeader,
+    /// The flash bank where the ISFB is located.
+    pub bank: u8,
+    /// The info flash page where the ISFB is located.
+    pub page: u8,
+    /// Padding for alignment.
+    #[serde(default)]
+    #[annotate(format=hex)]
+    pub pad: u16,
+
+    /// The erase policy for the ISFB region.
+    #[annotate(format=hex)]
+    pub erase_conditions: u32,
+    /// The key domain associated with the erase condition policy.
+    #[annotate(format=hex)]
+    pub key_domain: u32,
```

Comment:
`pub key_domain: ApplicationKeyDomain`

There are pre-defined constants for `Prod`, `Dev` and `Test`.  The `ApplicationKeyDomain` type also permits arbitrary integer values in addition to the well-known enumerated values.  See `ownership/application_key.rs`.

Reply:
Thanks. Updated.